src/app/modules/user/components/project-page/project-page.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="projectPage">
+    <div class="header">
+        <div class="header__title">
+            <h1>{{titleProject}}</h1>
+            <div class="header__splitter"></div>
+            <p>{{descriptionProject}}</p>
+        </div>
+        <div class="header__menu">
+            <a mat-flat-button class="header__button">Редактировать</a>
+            <button mat-icon-button><mat-icon>more_vert</mat-icon></button> <!-- кнопочка не отображается, не понял почему( --> 
+        </div>
+    </div>
+    <hr>
+    <div class="content">
+        <div class="details">
+            <h4>Детали проекта</h4>
+            <div class="description">
+                <th class="description__name">
+                    <tr>Заказчик</tr>
+                    <tr>Статус</tr>
+                    <tr>Запуск проекта</tr>
+                    <tr>Проект длится</tr>
+                    <tr>Команда</tr>
+                </th>
+                <th class="description__text">
+                    <tr *ngFor="let el of detailsProject">{{el}}</tr>
+                </th>
+                <div class="time">
+                    <h4>Прогресс команды</h4>
+                    <div class="time__elements">
+                        <div>
+                            <h1>264 ч</h1>
+                            <p>Июнь 2020</p>
+                        </div>
+                        <div>
+                            <h1>2858 ч</h1>
+                            <p>За все время</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="team">
+            <h4>Команда</h4>
+            <div class="team__content">
+                <div class="leaders">
```

Comment:
I think you can use * ngRor to output the team leaders. You will get an array with members from the back-end, this array will have a field with the role in the team, you can use * ngIf to filter by role.